pypeit/spectrographs/gtc_osiris.py

```diff
@@ -116,6 +118,10 @@ def default_pypeit_par(cls):
         par['calibrations']['arcframe']['exprng'] = [None, None]  # Long arc exposures
         par['calibrations']['arcframe']['process']['clip']=False
         par['calibrations']['arcframe']['process']['combine']='mean' #Multiple arcs with different lamps, so can't median combine
+        par['calibrations']['arcframe']['process']['use_continuum']=True #Multiple arcs with different lamps, so can't median combine
+        par['calibrations']['tiltframe']['process']['clip']=False
+        par['calibrations']['tiltframe']['process']['combine']='mean' #Multiple arcs with different lamps, so can't median combine
```

Comment:
Please update the comments here and below

Reply:
Done!